doc/optimization/multi_seeding.rst

```diff
@@ -0,0 +1,10 @@
+Using multiple seeds
+====================
+
+A seed can be defined by implementing the :py:meth:`OptimizaionProblem.seed` method.
```

Comment:
There's a typo in `OptimizaionProbelm.seed`

Reply:
Yep, will fix this.